cli/asc.js

```diff
@@ -1041,7 +1041,8 @@ exports.main = function main(argv, options, callback) {
     try {
       stats.writeCount++;
       stats.writeTime += measure(() => {
-        mkdirp(path.join(baseDir, path.dirname(filename)));
+        const dirPath = path.join(baseDir, path.dirname(filename));
```

Comment:
I think the problem is not the creation, but rather the `join`. I think this would be the fix here:
```suggestion
        const dirPath = path.resolve(baseDir, path.dirname(filename));
```

This way, the following behavior emerges:

- `dirname(filename) === "tmp"` -> `dirPath = baseDir + "/tmp"`
- `dirname(filename) === "./tmp"` -> `dirPath = baseDir + "/tmp"`
- `dirname(filename) === "/tmp"` -> `dirPath = "/tmp"`

Reply:
Makes sense. Could you try with latest commit?